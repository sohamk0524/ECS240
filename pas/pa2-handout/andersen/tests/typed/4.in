c Test case with address-of statements and single deref on LHS and RHS
p 9 9
t 1 2
t 2 2
t 3 2
t 4 1
t 5 1
t 6 1
t 7 1
t 8 0
t 9 0
c layer 2
s 0 1 -1 4
s 0 2 -1 5
s 0 2 -1 6
s 0 3 -1 7
c layer 1
s 0 5 -1 8
s 0 6 1 3
s 1 1 0 5
s 1 2 1 2
s 1 3 -1 9
