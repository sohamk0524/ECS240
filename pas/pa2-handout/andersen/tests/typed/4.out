pt 1 4
pt 2 5
pt 2 6
pt 3 7
pt 4 8
pt 5 8
pt 6 9
pt 7 9
